pkg/rules/slp086.go

```
@@ -0,0 +1,164 @@
+				}
+			}
+
+			// Second pass: determine route boundaries using brace depth
+			for i, route := range routes {
+				depth := 0
+				routeStarted := false
+				for j := route.startIdx; j < len(h.Lines); j++ {
+					ln := h.Lines[j]
+					// Count braces in this line
+					for _, ch := range ln.Content {
+						if ch == '(' || ch == '{' {
```

golangci-lint (2.11.4)

[error] QF1003: could use tagged switch on ch

(staticcheck)